spec/ttml1.xml

```diff
@@ -4598,12 +4598,19 @@ as a style property only to those element types indicated in the following table
 computed value of the style property must be considered to be no less than the
 size of the largest font size that applies to the element and its
 descendant elements in the intermediate synchronic document as determined by
-<specref ref="semantics-region-layout-step-1"/>. However, the value <code>normal</code> is not relative,
-and is therefore inherited as-is, as specified at <specref ref="semantics-style-resolution-processing-css"/>.</p>
+<specref ref="semantics-region-layout-step-1"/>. Furthermore,
+when performing <specref ref="semantics-style-resolution-processing-css"/>, step (4), [relative value resolution],
+the value <code>normal</code> is considered to be <emph>relative</emph> only on the most nested elements to which this property applies, namely,
+on <el>p</el> elements; otherwise, it is not considered to relative.</p>
 <note role="elaboration">
-<p>If the value of this attribute is <code>normal</code>, it is recommended that presentation processors
-use a line height that is 125% of the size of the largest applicable font size.
-The choice of 125% allows use of a font size of <code>0.8c</code> in order
+<p>Since the value <code>normal</code> is considered to be non-relative for all ancestors of <el>p</el>,
+this value is inherited as is without further resolution; however, when the computed style set is obtained on
+a <el>p</el> element, the value <code>normal</code> is treated as a relative value, and, therefore, is resolved
+to an absolute form during [relative value resolution].</p></note>
+<note role="elaboration">
+<p>It is recommended that presentation processors use a computed line height that is
```

Comment:
I don't understand why this sentence has been reordered, and it seemed better before. Beginning with the condition helps define the scope better. "If this then that" is a good pattern.